Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -468,7 +468,10 @@ public boolean onCreateOptionsMenu(Menu menu)
 	@Override
 	public boolean onOptionsItemSelected(MenuItem item)
 	{
-		handleMenuAction(buttonsActionsMap.get(item.getItemId()));
+		if (buttonsActionsMap.get(item.getItemId(), -1) >=0)
```

Comment:
I'm not familiar with the coding style here but it probably needs a space after ">=".